docs/reference/commandline/run.md

```diff
@@ -479,3 +479,18 @@ available in the default container, you can set these using the `--ulimit` flag.
 
 The values are sent to the appropriate `syscall` as they are set.
 Docker doesn't perform any byte conversion. Take this into account when setting the values.
+
+#### For `nproc` usage:
+
```

Comment:
No need to start with a note.  Also, some awkward ESL errors.

--->  https://gist.github.com/moxiegirl/5acc5b68600bb80a6b1d

#### For `nproc` usage:

Be careful setting `nproc` with the `ulimit` flag as `nproc` is designed by Linux to set the maximum number of processes available to a user, not to a container.  For example, start four containers with `daemon` user:


    docker run -d -u daemon --ulimit nproc=3 busybox top
    docker run -d -u daemon --ulimit nproc=3 busybox top
    docker run -d -u daemon --ulimit nproc=3 busybox top
    docker run -d -u daemon --ulimit nproc=3 busybox top // fail

The 4th container fails and reports "No resource available" error. This fails because the caller 
set `nproc=3` resulting in the first three containers using up the three processes quota set for the `daemon` user.